src/F0_output.py

```diff
@@ -3,11 +3,32 @@
 import pandas as pd
 import logging
 import matplotlib.pyplot as plt
+import os
 
 import src.F1_plotting as plots
 
+r"""
+Module F0 Output
+================
+The model F0 output defines all functions that store evaluation results to file.
+- Aggregate demand profiles to a total demand profile
+- Plotting all energy flows for both 14 and 365 days for each energy bus
```

Comment:
```suggestion
- Plot all energy flows for both 14 and 365 days for each energy bus
```

Reply:
You can add all this suggestion to batch (you must be in the "Files changed" tab to do so) to group them into one commit :)